azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@
 trigger:
   branches:
     include:
-    - development
```

Comment:
It would be nice to have here a solution which is working for all branches. It is very annoying when you need to add the branch every time you create it

Reply:
Now I can just name a new branch "b/" with the name followed, and it should trigger a build